drawBot/context/baseContext.py

```diff
@@ -1894,7 +1894,7 @@ def fontNamedInstance(self, name: str, fontNameOrPath: str | SomePath | None = N
                 fontName = self._font
             raise DrawBotError(f"Can not find instance with name: '{name}' for '{fontName}'.")
 
-    def listNamedInstances(self, fontNameOrPath: SomePath | None = None, fontNumber: int = 0) -> dict[str, dict]:
+    def listNamedInstances(self, fontNameOrPath: SomePath | None = None) -> dict[str, dict]:
```

Comment:
Is it really necessary to drop the `fontNumber` argument? That would be an unfortunate change.

Reply:
can variable fonts be inside .ttc?

Reply:
Yes

Reply:
oke, didnt know, will add fontNumber back